lib/fcm.rb

```diff
@@ -49,8 +49,8 @@ def create_notification_key(key_name, project_id, registration_ids = [])
       'project_id' => project_id
     }
 
-    for_uri(GROUP_NOTIFICATION_BASE_URI, extra_headers) do |connection|
-      response = connection.post('/gcm/notification', post_body.to_json)
+    for_uri(BASE_URI, extra_headers) do |connection|
```

Comment:
@Alan-M-Thomaz

since there are four places where we need to call ```/fcm/notification```

- create_notification_key(post)
- add_registration_ids(post)
- remove_registration_ids(post)
- recover_notification_key(get)

maybe it would be better if we refactor this api call into a separate private method, like what ```execute_notification``` method does..

```ruby
def update_group_messaging_setting(body)
  for_uri(BASE_URI, extra_headers) do |connection|
      response = connection.post('/fcm/notification', body.to_json)
      build_response(response)
  end
end
```

Reply:
done, except for `recover_notification_key`, because it's a get request